tools/build/citool

```diff
@@ -209,14 +209,18 @@ def grepForFailingTests(args, body):
             print(output.replace('&gt;', '>')),
 
 def reportBuildStatus(args, body):
-    cmd = 'tail -1'
-    (time, output, error) = shell(cmd, body, args.verbose)
-    output = output.replace('</span>', '').replace('</a>', '')
-    if 'Finished: ' in output or output.startswith('Done.'):
-        print output.strip()
+    lines = body.rstrip('\n').rsplit('\n', 1)
+    if len(lines) == 2:
+        output = lines[1]
+        output = re.sub('<[^<]+?>', '', output).strip()
```

Comment:
http://stackoverflow.com/questions/1732348/regex-match-open-tags-except-xhtml-self-contained-tags/1732454#1732454 (some light reading, not an actual suggestion)